config/deploy/kubernetes/kubernetes-all.yaml

```diff
@@ -3100,54 +3100,6 @@ status:
   conditions: []
   storedVersions: []
 ---
-# Source: dynatrace-operator/templates/Common/activegate/serviceaccount-activegate.yaml
-# Copyright 2021 Dynatrace LLC
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-
-#     http://www.apache.org/licenses/LICENSE-2.0
-
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-apiVersion: v1
-kind: ServiceAccount
```

Comment:
we still need ServiceAccount, ClusterRoles and ClusterRoleBIndings for the Activegate.

Reply:
fixed in [a0a7c36](https://github.com/Dynatrace/dynatrace-operator/pull/745/commits/a0a7c36fe204520278566272387347f366689ae8)